Makefile

```diff
@@ -49,6 +49,9 @@ salus:
 
 release: add-license shorten-lines format test lint salus
 
+compile:
+	xgo --targets=darwin/*,windows/*,linux/* -out bin/rosetta-cli-$(version) .
```

Comment:
Is it possible to specify a tag here or do we need to checkout the code we want to compile @juliankoh ?

Reply:
This builds the code locally (as you can see from the . )

We can also build a specific tag but that would be a different command.